document-generator-accounts/src/main/java/uk/gov/companieshouse/document/generator/accounts/AccountsDocumentInfoServiceImpl.java

```diff
@@ -43,11 +49,48 @@ public DocumentInfo getDocumentInfo() {
                 // company accounts should work for abridged, resulting in this abridged specific code
                 // qualifying for removal
                 .filter(this::isAbridged)
-                .map(accountsLinks -> new DocumentInfo())
+                .map(accountsLinks -> getAbridgedAccountData(resourceId))
                 .orElse(null);
 
     }
 
+    /**
+     * Gets the abridged account data from abridged specific services (abridged-api).
+     * @param resourceId - the abridged account resource link
+     * @return DocumentInfo object containing information requested from document-generator-core
+     */
+    private DocumentInfo getAbridgedAccountData(String resourceId) {
+        Accounts accounts = accountsService.getAccounts(resourceId);
+
+        if (accounts == null) {
+            return null;
+        }
+
+        AccountType accountsType = getAccountType(accounts);
+        if (accountsType == null) {
+            return null;
+        }
```

Comment:
Why is this code in an abridged specific method? There's nothing specific to abridged in here.